g11n-ws/modules/md-service-i18n-l2/src/main/java/com/vmware/i18n/l2/service/locale/LocaleService.java

```diff
@@ -38,6 +38,8 @@
 	private static final String UI_LIST_OR_MENU = "uiListOrMenu";
 	private static final String NO_CHANGES = "no-change";
 	private static final Logger logger = LoggerFactory.getLogger(LocaleService.class.getName());
+	private static final String TERRITORY_REGIONS = "terr-region";
+	private static final String TERRITORY_CITYS = "terr-city";
```

Comment:
How about put cities data in a separate cache?

Reply:
Use the different cache key prefix

Reply:
To reduce the cache key size